src-tauri/src/commands.rs

```diff
@@ -0,0 +1,1511 @@
+use crate::app_config::{AppConfig, GpuThreads};
+use crate::app_in_memory_config::AirdropInMemoryConfig;
+use crate::auto_launcher::AutoLauncher;
+use crate::binaries::{Binaries, BinaryResolver};
+use crate::credential_manager::{CredentialError, CredentialManager};
+use crate::external_dependencies::{
+    ExternalDependencies, ExternalDependency, RequiredExternalDependency,
+};
+use crate::gpu_miner_adapter::GpuNodeSource;
+use crate::hardware::hardware_status_monitor::HardwareStatusMonitor;
+use crate::internal_wallet::{InternalWallet, PaperWalletConfig};
+use crate::node_manager::NodeManagerError;
+use crate::p2pool::models::Stats;
+use crate::progress_tracker::ProgressTracker;
+use crate::systemtray_manager::{SystemtrayManager, SystrayData};
+use crate::tor_adapter::TorConfig;
+use crate::utils::shutdown_utils::stop_all_processes;
+use crate::wallet_adapter::TransactionInfo;
+use crate::wallet_manager::WalletManagerError;
+use crate::{
+    setup_inner, ApplicationsVersions, BaseNodeStatus, CpuMinerMetrics, GpuMinerMetrics,
+    MaxUsageLevels, MinerMetrics, TariWalletDetails, UniverseAppState, APPLICATION_FOLDER_ID,
+    MAX_ACCEPTABLE_COMMAND_TIME,
+};
```

Comment:
Remove those imports and paste these struct directly in this file. In a case of `ApplicationsVersions` it's declared in `main.rs` but used only in `commands.rs`. It can be simplified by simply declaring it here.